src/CNGLOB.H

```diff
@@ -178,6 +178,21 @@ typedef std::string WStr;
 						//   NOTE: < > required re rcdef build
 
 /*---------------------- global macros and defines -------------------------*/
+// useful constants
+const double kPi=3.14159265358979323846;
```

Comment:
I'm curious why you had to relocate the constants, and then, while we're at it, does it make sense to make them constexpr?

Reply:
I had to relocate them because one of the variables was used before it was declared. Yes, I think it meets all the requirements to make them constexpr.

Reply:
Could you post the error you got? I'm a little confused as to why it worked all this time and now doesn't.

Reply:
I will. To clarify it works on msvc, it just doesn't work on POSIX system.

Reply:
<img width="1440" alt="Screen Shot 2022-05-27 at 11 56 56 AM" src="https://user-images.githubusercontent.com/17099469/170765366-b4c7209f-165e-4ed9-9fae-310f9386aa1d.png">